### PR TITLE
Provide factory to map from (coffee, notation) to workflow model

### PR DESCRIPTION
**Pull request by martin-fleck-at**
- Provide mapping for coffee example to workflow model elements
- Provide convenience builder for workflow model elements
- Use builder in factory and in creation handlers

Please note this PR depends on https://github.com/eclipsesource/graphical-lsp/pull/307 so the changes look far bigger than what I actually did (for now) ;-)

### CODE REVIEW

File: server/example/workflow-modelserver-example/src/main/java/com/eclipsesource/glsp/example/modelserver/workflow/model/CoffeeTypeUtil.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+import com.eclipsesource.modelserver.coffee.model.coffee.ManualTask;
+import com.eclipsesource.modelserver.coffee.model.coffee.Merge;
+import com.eclipsesource.modelserver.coffee.model.coffee.Node;
+
+public final class CoffeeTypeUtil {	
+	public static final String FORK_NODE = "activityNode:fork";
+	public static final String JOIN_NODE = "activityNode:join";
+	
+	public static String toType(Node node) {
+		if(node instanceof AutomaticTask) {
```
Comment (planger) [inline server/example/workflow-modelserver-example/src/main/java/com/eclipsesource/glsp/example/modelserver/workflow/model/CoffeeTypeUtil.java]: There are a few formatting issues (the "(" has no space before the if).

File: server/example/workflow-modelserver-example/src/main/java/com/eclipsesource/glsp/example/modelserver/workflow/model/WorkflowModelServerModelFactory.java [hunk 2]
```diff
@@ -15,23 +15,153 @@
...
+		// 1. Load models and create workflow facade
+		String sourceURI = action.getOptions().get(ClientOptions.SOURCE_URI);
+		WorkflowModelServerAccess modelAccess = new WorkflowModelServerAccess(sourceURI);
 		WorkflowFacade workflowFacade = modelAccess.getWorkflowFacade();
+		if(workflowFacade == null) {
+			return createEmptyRoot();
+		}
+		
+		// 2. Check if request model action can be fulfilled by checking which workflow to load
+		// TODO: Add parameter to RequestModelAction to determine which workflow to use
```
Comment (planger) [inline server/example/workflow-modelserver-example/src/main/java/com/eclipsesource/glsp/example/modelserver/workflow/model/WorkflowModelServerModelFactory.java]: I'm wondering whether we should rather put the workflow to use into the client options.

File: server/example/workflow-modelserver-example/src/main/java/com/eclipsesource/glsp/example/modelserver/workflow/model/CoffeeTypeUtil.java (hunk 1 above)
Comment (martin-fleck-at) [reply server/example/workflow-modelserver-example/src/main/java/com/eclipsesource/glsp/example/modelserver/workflow/model/CoffeeTypeUtil.java]: Very good point, I ran the formatter in all changed files again to be sure everything is correctly formatted!

File: server/example/workflow-modelserver-example/src/main/java/com/eclipsesource/glsp/example/modelserver/workflow/model/WorkflowModelServerModelFactory.java (hunk 2 above)
Comment (martin-fleck-at) [reply server/example/workflow-modelserver-example/src/main/java/com/eclipsesource/glsp/example/modelserver/workflow/model/WorkflowModelServerModelFactory.java]: Yes, you are right, that was badly phrased, we should check the options of course. I added option handling in a follow-up commit.

**Pull request merged**